src/ListGroup.js

```diff
@@ -35,6 +35,10 @@ class ListGroup extends React.Component {
     }
   }
 
+  isAnchor(props){
+    return (props.href || props.target || props.onClick);
```

Comment:
`target` by itself wouldn't do anything. Probably best to exclude from this check?